### ISSUE TITLE
Magento migration script doesn't honor system paths for wget

### ISSUE THREAD

**Issue by djixas** [OpenMage/magento-lts#2833]
### Preconditions (*)

1. Installed Magento Commuinity Edition 1.9.4.5
2. Downloading patch for Magento CE 1.9.4.5 to OpenMage LTS v19.4.8

### Steps to reproduce (*)

1. Run $ curl -fsSL https://migrate.openmage.org | sh import script via user while main server installation has other wget version

### Expected result (*)

1. It should honor paths and not pick defaults

### Actual result (*)

1. wget: unrecognized option '--no-hsts'
Usage: wget [OPTION]... [URL]...
Try `wget --help' for more options._

Hi, as you may know, CentOS has older wget which makes openmagelts script not compatible.

I've installed wget as separate bin, but it seems there is no way to "force' a different path for script. Even if I install wget 1.21, OpenMage script does not honor user wget version. It relies on default paths. I did login as user on separate shell account, which gives correct version:

[xx@host ~/public_html]# wget --version
GNU Wget 1.21 built on linux-gnu.

But that does not matter to openmage. The script is simply relying on the system default paths for its dependencies, resulting in wget error since default centos version is 1.14.

**Comment by djixas** [comment]
On the other hand it looks like CentOS may not actually require its own version of wget anyway so I have removed the version provided by CentOS and linked the default path (/bin/wget) to the newer version I installed manually in /opt.

The script is now working in that regard, but it's failing elsewhere when it applies patches to the Magento installation inside the public_html/ directory:

Performing dry-run patch...
warning: .htaccess has type 100755, expected 100644
error: patch failed: .htaccess:1
error: .htaccess: patch does not apply
error: .htaccess.sample: No such file or directory
warning: app/etc/config.xml has type 100755, expected 100644
Unable to apply the patch. The following command failed:
    git --git-dir=/dev/null apply --ignore-whitespace --check var/magento-ce-1.9.4.5-openmage-lts-v19.4.8.patch


However the developers of the migration script should look into this further as it appears to be attempting to apply a specific "openmage" patch.
